h20/pubspec.yaml

```diff
@@ -59,9 +60,8 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  # assets:
-  #   - images/a_dot_burr.jpeg
-  #   - images/a_dot_ham.jpeg
+  assets:
+    - images/
```

Comment:
Never make separate folders for images/fonts/animation at the root level. Create an assets folder that will contain all types of child folders by type.

Reply:
I was aware of that information, but I simply forgot while working on this project. Thanks.